odfuzz/entities.py

```diff
@@ -144,7 +144,7 @@ def _append_queryable(self, query_group_data):
         # TODO REFACTOR DRY this method is direct copypaste from DispatchedBuilder just to have a prototype for integration. Intentionally no abstract class at the moment.
         if Config.fuzzer.http_method_enabled != "POST":
             self._append_corresponding_queryable(QueryGroupSingle(query_group_data))
-        if Config.fuzzer.http_method_enabled != "PUT":
+        if Config.fuzzer.http_method_enabled != "PUT" and Config.fuzzer.http_method_enabled != "MERGE":
```

Comment:
I think this should be OR logic, not AND. 

You can have only one method generation enabled in the config and it is expected that will work, unrelated to the other method.

Reply:
I totally misunderstood the logic and purpose due inverted != logic. 
Agreed in person to rewrite this part to normal IF a == b logic, so the method is more readable and maintainable when we add new HTTP method support

Reply:
addressed